Dockerfile

```
@@ -17,9 +17,13 @@ ENV SYSTEM_USER_GID ${NB_GID}
 ENV PYTHONPATH /home/$SYSTEM_USER
 ENV CONDA_DIR /opt/conda
 ENV PATH $CONDA_DIR/bin:$PATH
-ENV MINICONDA_VERSION 4.7.12.1
-ENV MINICONDA_MD5 81c773ff87af5cfac79ab862942ab6b3
-ENV CONDA_VERSION 4.8.2
+
+# Modify this section for the conda/python update.
+# This list of miniconda installer versions together with their SHA256 check sums are available:
+# https://docs.conda.io/en/latest/miniconda_hashes.html
+ENV MINICONDA_VERSION py37_4.9.2
+ENV MINICONDA_SHA256 79510c6e7bd9e012856e25dcb21b3e093aa4ac8113d9aa7e82a86987eabe1c31
+ENV CONDA_VERSION 4.9.2
```

How about something this to make it easier to extract the individual versions?

```suggestion
ENV PYTHON_VERSION py37
ENV CONDA_VERSION 4.9.2
ENV MINICONDA_VERSION $PYTHON_VERSION_$CONDA_VERSION
ENV MINICONDA_SHA256 79510c6e7bd9e012856e25dcb21b3e093aa4ac8113d9aa7e82a86987eabe1c31
```

---

looks good to me. Of course, I can't promise that they will keep using this naming strategy in the future. But we will figure it out in case they won't ;)